3684/0/about_control_statements.rb

```diff
@@ -16,17 +23,27 @@ def test_if_then_statements
     assert_equal :true_value, result
   end
 
+  # rubocop:enable Metrics/ClassLength
+  # rubocop:disable Metrics/MethodLength
```

Comment:
Please explain why you have to disable this cop here

Reply:
this comments left after I checked code with rubocop 0.72.0, so methods below and above require this disables, but I surrond them separatly, and forgot to clean this 2 lines

Reply:
please remove the unnecessary cops `disablers`